src/test/java/org/prebid/server/bidder/algorix/AlgorixBidderTest.java

```diff
@@ -0,0 +1,266 @@
+package org.prebid.server.bidder.algorix;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.request.Imp;
+import com.iab.openrtb.request.Format;
+import com.iab.openrtb.request.Banner;
+import com.iab.openrtb.request.Video;
+import com.iab.openrtb.request.Native;
+import com.iab.openrtb.response.Bid;
+import com.iab.openrtb.response.BidResponse;
+import com.iab.openrtb.response.SeatBid;
+import org.junit.Before;
+import org.junit.Test;
+import org.prebid.server.VertxTest;
+import org.prebid.server.bidder.model.HttpRequest;
+import org.prebid.server.bidder.model.HttpCall;
+import org.prebid.server.bidder.model.HttpResponse;
+import org.prebid.server.bidder.model.Result;
+import org.prebid.server.bidder.model.BidderBid;
+import org.prebid.server.bidder.model.BidderError;
+import org.prebid.server.proto.openrtb.ext.ExtPrebid;
+import org.prebid.server.proto.openrtb.ext.request.algorix.ExtImpAlgorix;
+
+import java.util.List;
+import java.util.function.Function;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.singletonList;
+import static java.util.function.Function.identity;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+import static org.prebid.server.proto.openrtb.ext.response.BidType.banner;
+import static org.prebid.server.proto.openrtb.ext.response.BidType.video;
+import static org.prebid.server.proto.openrtb.ext.response.BidType.xNative;
+
+/**
+ * AlgoriX Bidder Test
+ * @author xunyunbo@algorix.co
+ */
+public class AlgorixBidderTest extends VertxTest {
+
+    private static final String ENDPOINT_URL = "https://xyz.svr-algorix.com/rtb/sa?sid={SID}&token={TOKEN}";
+    private static final String IMP_ID = "123";
+    private static final int FORMAT_W = 320;
+    private static final int FORMAT_H = 50;
```

Comment:
We prefer not to create constants in test, just use them directly in test methods